addon/routes/fd-editform-constructor.js

```diff
@@ -88,99 +94,267 @@ export default Ember.Route.extend({
-
-                propertyOwnerId = relationship.get('startClass.id');
-              }
+          let propertyDefinition = definition[i];
+          if (!propertyDefinition.visible) {
+            continue;
+          }
 
-              if (skipAsExternalClass) {
-                attributes.popObject();
-                continue;
-              }
+          _this._locateControl(controlTree, propertyDefinition, propertyDefinition.path);
+        }
 
-              if (propertyOwner.get('id') !== propertyOwnerId) {
-                propertyOwner = this.get('model.dataObjects').findBy('id', propertyOwnerId);
-              }
+        modelHash.controls = controlTree;
+        resolve(modelHash);
+      });
+    });
+  },
 
-              let classAttribute = propertyOwner.get('attributes').findBy('name', attributeName);
+  /**
+      Locate control by path.
 
-              if (!classAttribute) {
-                classAttribute = this._getAttributeFromParent(propertyOwnerId, attributeName);
-              }
+      @method _locateControl
+      @param {Object} controlTree Controls tree.
+      @param {FdDefinition} propertyDefinition Property definition from view.
+      @param {String} path Property path from view.
+      @return {Object} Modified controlTree with placed control.
+  */
+  _locateControl: function (controlTree, propertyDefinition, path) {
+    if (!path || path === '') {
+      let row  = FdEditformRow.create({ controls: Ember.A(), columnsCount: 0 });
+
+      if (controlTree.rows) {
+        controlTree.rows.pushObject(row);
+      } else {
+        controlTree.pushObject(row);
+      }
+
+      this._locateControlInRow(row, propertyDefinition);
+
+      return controlTree;
+    }
+
+    if (path.charAt(0) === '|') {
+      return this._locateTabs(controlTree, propertyDefinition, path);
+    }
 
-              if (classAttribute) {
-                attribute.classAttribute = classAttribute;
-                attribute.type = classAttribute.get('type');
-                attribute.notNull = classAttribute.get('notNull');
-                attribute.defaultValue = classAttribute.get('defaultValue');
+    if (path.charAt(0) === '-') {
+      return this._locateGroup(controlTree, propertyDefinition, path);
+    }
+
+    if (path.charAt(0) === '#') {
+      return this._locateColumn(controlTree, propertyDefinition, path);
+    }
+
+    // TODO: Show error and fix it.
+  },
+
+  /**
+      Locate control in row by path.
+
+      @method _locateControlInRow
+      @param {FdEditformRow} row Row for control placement.
+      @param {FdDefinition} propertyDefinition Property definition from view.
+      @return {Object} Modified controlTree with placed control.
+  */
+  _locateControlInRow: function (row, propertyDefinition) {
+    // TODO: вычислить type контрола из метаданных атрибута или FormControl и width из path.
+    let control = FdEditformControl.create({ caption: propertyDefinition.name, type: 'string', width: '100*' });
```

Comment:
Need to use the [`caption`](https://github.com/Flexberry/ember-flexberry-designer/blob/feature-165152-route-model-fd-editform-constructor/addon/objects/fd-view-attributes-property.js#L29) property instead of `name`.